tests/articulation_tests.ts

```diff
@@ -279,7 +280,7 @@ function drawArticulations2(options: TestOptions, contextBuilder: ContextBuilder
 
 function tabNotes(options: TestOptions, contextBuilder: ContextBuilder): void {
   const ctx = contextBuilder(options.elementId, 600, 200);
-  ctx.font = '10pt Arial';
+  ctx.font = '10pt ' + Font.SANS_SERIF;
```

Comment:
I think my modifications to the tests should be left out of this PR. In my current version, I have left comments indicating which lines will introduce visual diffs, and then I have reverted to the original code (e.g., '10pt Arial').

Reply:
This one is causing no visual diff and you left `Font.SANS_SERIF`. Is it OK to leave it or you want it reverted?

Reply:
Approved, no change required.

Reply:
Yes, leave it as is. I'll deal with the rebase and fix any conflicts if they arise.